Use US standards by default where imperial or metric alternatives exist.

diff --git units.c units.c
index 3d09ba0..5253ef4 100644
--- units.c
+++ units.c
@@ -36,7 +36,7 @@ typedef struct _UnitTable
 UnitTable unit_table [] = 
   { 
   { atmosphere, "atmosphere", "atm,atmospheres", "atmosphere" ,"atmospheres" }, 
-  { acre, "acre", "acres", "acre (international)", "acres" }, 
+  { acre, "siacre", "siacres", "acre (international)", "siacres" }, 
   { angstrom, "angstrom", "angstroms,ang,angs", "angstrom", "angstroms" }, 
   { ampere, "ampere", "amperes,amp,amps,A", "ampere", "amperes" }, 
   { au, "AU", "au", "astronomical unit", "AUs" }, 
@@ -73,7 +73,7 @@ UnitTable unit_table [] =
   { foot, "foot", "feet,ft", "imperial foot" , "feet" }, 
   { footcandle, "footcandle", "footcandles,fc", "footcandle" , "footcandles" }, 
   { footlambert, "footlambert", "footlamberts,fL", "footlambert" , "footlamberts" }, 
-  { gallon, "gallon", "gallons,gal", "imperial gallon" , "gallons" }, 
+  { gallon, "ukgallon", "ukgallons,ukgal", "imperial gallon" , "ukgallons" }, 
   { gibibit, "gibibit", "gibibits,gibit,gibits", "gibibits", "gibibits" },
   { gibibyte, "gibibyte", "gibibytes,gib", "gibibytes", "gibibytes" },
   { gigabit, "gigabit", "gigabits,gbit,gbits", "gigabits", "gigabits" },
@@ -88,8 +88,8 @@ UnitTable unit_table [] =
   { hectare, "hectare", "ha,hectares", "hectare" , "hectares" }, 
   { hour, "hour", "hours,h,hr,hrs", "hour" , "hours" }, 
   { horsepower, "horsepower", "hp", "horsepower (metric)" , "horsepower" }, 
-  { hundredweight, "hundredweight", "cwt", "impl. hundredweight" , 
-     "hundredweight" }, 
+  { hundredweight, "ukhundredweight", "ukcwt", "impl. hundredweight" , 
+     "uk hundredweight" }, 
   { inch, "inch", "inches,in", "imperial inch" , "inches" }, 
   { joule, "joule", "joules,j,J", "joule" , "joules" }, 
   { kelvin, "kelvin", "kelvins,k,K", "kelvin" , "kelvin" }, 
@@ -120,7 +120,7 @@ UnitTable unit_table [] =
   { nauticalmile, "nautical-mile", "nautical mile,nmi,NM", 
     "nautical mile" , "nautical-miles" }, 
   { mmHg, "mmHg", "mmhg", "mm of mercury" , "mmHg" }, 
-  { mpg, "mpg", "mpg", "miles per gallon (UK)" , "mpg" }, 
+  { mpg, "ukmpg", "ukmpg", "miles per gallon (UK)" , "ukmpg" }, 
   { mph, "mph", "mph", "miles per hour" , "mph" }, 
   { newton, "newton", "newtons,N", "newton" , "newtons" }, 
   { ounce, "ounce", "ounces,oz", "avoirdupois ounce" , "ounces" }, 
@@ -130,13 +130,13 @@ UnitTable unit_table [] =
   { petabit, "petabit", "petabits,pbit,pbits", "petabits", "petabits" },
   { petabyte, "petabyte", "petabytes,pb", "petabytes", "petabytes" },
   { phot, "phot", "phots,ph", "phot" , "phots" }, 
-  { pint, "pint", "pints,pt,pts", "imperial pint" , "pints" }, 
+  { pint, "ukpint", "ukpints,ukpt,ukpts", "imperial pint" , "ukpints" }, 
   { point, "point", "points", "type point (US, Eng)" , "points" }, 
   { pound, "pound", "pounds,lb,lbs", "imperial pound" , "pounds" }, 
   { poundforce, "pound-force", "poundforce,poundsforce,pounds-force,lbf", 
      "imperial pound-force", "pounds-force" }, 
   { psi, "psi", "PSI", "pounds per sq inch" , "psi" }, 
-  { quart, "quart", "quarts,qt,qts", "imperial quart" , "quarts" }, 
+  { quart, "ukquart", "ukquarts,ukqt,ukqts", "imperial quart" , "ukquarts" }, 
   { rad, "rad", "rads", "rad" , "rads" }, 
   { radian, "radian", "radians", "angular radian" , "radians" }, 
   { REM, "REM", "REMS,rem,rems", "Roentgen equiv. man"  , "REM" }, 
@@ -153,7 +153,7 @@ UnitTable unit_table [] =
   { terabit, "terabit", "terabits,tbit,tbits", "terabits", "terabits" },
   { terabyte, "terabyte", "terabytes,tb", "terabytes", "terabytes" },
   { therm, "therm", "therms", "therm" , "therms" }, 
-  { ton, "ton", "tons", "imperial (long) ton" , "tons" }, 
+  { ton, "ukton", "uktons", "imperial (long) ton" , "uktons" }, 
   { ton_tnt, "ton-tnt", "tons-tnt,ttnt", "ton of TNT (energy)" , "tons-tnt" }, 
   { tonne, "tonne", "tonnes", "metric tonne" , "tonnes" }, 
   { torr, "torr", "torrs", "torr" , "torrs" }, 
@@ -161,14 +161,14 @@ UnitTable unit_table [] =
     "troy-ounces" }, 
   { troy_pound, "troy-pound", "troy-pounds,troy pound,troy pounds,lbt", "troy pound" , 
     "troy-pounds" }, 
-  { usacre, "usacre", "usacres", "US acre" , "usacres" }, 
-  { usgallon, "usgallon", "usgallons,usgal,usgals", "US gallon" , "usgallons" }, 
-  { ushundredweight, "ushundredweight", "uscwt", "US hundredweight" , 
+  { usacre, "acre", "acres", "US acre" , "acres" }, 
+  { usgallon, "gallon", "gallons,gal,gals", "US gallon" , "gallons" }, 
+  { ushundredweight, "hundredweight", "cwt", "US hundredweight" , 
      "US hundredweight" }, 
-  { usmpg, "usmpg", "usmpg", "miles per gallon (US)" , "usmpg" },
-  { uspint, "uspint", "uspints,uspt,uspts", "US pint" , "uspints" }, 
-  { usquart, "usquart", "usquarts,usqt,usqts", "US quart" , "usquarts" }, 
-  { uston, "uston", "ustons", "US (short) ton" , "ustons" }, 
+  { usmpg, "mpg", "mpg", "miles per gallon (US)" , "mpg" },
+  { uspint, "pint", "pints,pt,pts", "US pint" , "pints" }, 
+  { usquart, "quart", "quarts,qt,qts", "US quart" , "quarts" }, 
+  { uston, "ton", "tons", "US (short) ton" , "tons" }, 
   { watt, "watt", "watts,w", "watt" , "watts" }, 
   { wood_cord, "wood-cord", "woodcord,woodcords,wood-cords", "wood cord (volume)"  ,
     "wood-cords" }, 
